### ISSUE TITLE
nodeid flag outputs extra line with new agents

### ISSUE THREAD

**Issue by wh1te909** [Ylianst/MeshCentral#3656]
Hi, maybe a question for @krayon007 

Before 0.9.81:
```
C:\Windows\system32>"c:\Program Files\Mesh Agent\MeshAgent.exe" -nodeid
E418C0836DB001F874EFB87BFB58A300237C98AD8FBE1741FDD217E4257FB9D1596BAC46ABB188FC9AD489BDAAE6B3BA
```
After: 
```
C:\Windows\system32>"c:\Program Files\Mesh Agent\MeshAgent.exe" -nodeid
E418C0836DB001F874EFB87BFB58A300237C98AD8FBE1741FDD217E4257FB9D1596BAC46ABB188FC9AD489BDAAE6B3BA
0
```

Just wanted to know if this is by design or a bug. If by design I will change my script to only get the first line of output (I use nodeid for integration to render in iframe)

Thanks!

**Comment by Ylianst** [comment]
This is a bug, totally my fault. I have known about this and will be releasing a new agent in the next few days to fix this.

**Comment by wh1te909** [comment]
haha appreciate it, thanks :)

**Comment by JSuenram** [comment]
> This is a bug, totally my fault. I have known about this and will be releasing a new agent in the next few days to fix this.

Is this already addressed? We are having the same issue....

**Comment by wh1te909** [comment]
this was fixed with the release of the new agents in 0.9.85, closing. thanks!
